gradle.properties

```diff
@@ -1 +1 @@
-version=5.1.5
+version=5.1.6-SNAPSHOT
```

Comment:
Oh NOOOOOOO!!!! 
`SNAPSHOT` again!

Reply:
Yes, just so the build doesn't fail as it tries to publish `5.1.5` a second time ;-) When this is complete and tested post-merge it'll be `5.1.6` don't worry :-)